nuxt/.env.template

```diff
@@ -1 +1,3 @@
-NUXT_PUBLIC_GOOGLE_ANALYTICS_ID=
+NUXT_TURNSTILE_SECRET_KEY=
+NUXT_TURNSTILE_SITE_KEY=
+HOST=
```

Comment:
Is this needed?

Reply:
If you mean the "HOST" variable as we spoke about before then I can say that it seems to me like we might need it. It is used in the nuxt.config.ts for the variable BASE_URL which is used for i18n. There was a comment saying that it was deprecated but that was removed in this [commit](https://github.com/maevsi/maevsi/commit/aec20ea2b7a80c4241ebec7d02f984a6543dd978). Maybe you know something more about this

Reply:
It should not be added without a specific reason, it could break things.

Reply:
I removed the `HOST` variable (fabbe8434fe574a8b68f6900f60a17a6956201ee)